R/derive_param_wbc_abs.R

```diff
@@ -0,0 +1,162 @@
+#' Add a parameter for lab differentials converted to absolute values
+#'
+#' @description
+#'
+#' Add a parameter by converting lab differentials from percentage to absolute values
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified by the `by_vars` argument, `PARAMCD`, and `AVAL`
+#'   are expected to be present.
+#'
+#'   The variable specified by `by_vars` and `PARAMCD` must be a unique key of
+#'   the input dataset, and to the parameters specified by `wbc_code` and `diff_code`.
+#'
+#' @param set_values_to Variables to set
+#'
+#'   A named list returned by `vars()` defining the variables to be set for the
+#'   new parameter, e.g. `vars(PARAMCD = "LYMPH", PARAM = "Lymphocytes Abs (10^9/L)")` is
+#'   expected.
+#'
+#' @param get_unit_expr An expression providing the unit of the parameter
+#'
+#'   The result is used to check the units of the input parameters.
+#'
+#'   Permitted Values: a variable containing unit from the input dataset, or a function call
+#'   (for example, `get_unit_expr = extract_unit(PARAM)`).
```

Comment:
Are the parentheses needed here?  
```suggestion
#'   , for example, `get_unit_expr = extract_unit(PARAM)`.
```

Reply:
Updated.